crates/rpc/src/v02/types/class.rs

```diff
@@ -402,8 +402,7 @@ pub struct SierraContractClass {
     pub sierra_program: Vec<Felt>,
     pub contract_class_version: String,
     pub entry_points_by_type: SierraEntryPoints,
-    #[serde(default)]
-    pub abi: Option<String>,
+    pub abi: String,
```

Comment:
This is breaking change of the `v02` API. Is it necessary in scope of this PR?

Reply:
It indeed is, as many other exceptions that are outlined in our readme. I'll add a proper node to the rpc section ofc. As for the change - it needs to stay, otherwise you won't be able to declare sierra classes. Btw 0.3 will make this field mandatory and in all honesty all those exceptions in v0.2 of ours should actually make it into a refined 0.2.2 which afaik will not happen :shrug: .

Reply:
Adding support for Broadcasted declare v2 in api v0.2 is mentioned in the readme. I don't think it really breaks the 0.2 api.